spec/requests/backend_crud_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe "managing backends", type: :request do
+  before :each do
+    User.create
+  end
```

Comment:
Rather than doing this for each test you could do: https://github.com/alphagov/content-store/blob/cf41056f3cee446ef94043f3a3b074c71bcfa7d6/spec/rails_helper.rb#L50-L52

Reply:
fixed